ui/app/registration/directives/patientRelationship.js

```diff
@@ -73,6 +73,14 @@ angular.module('bahmni.registration')
                 return personRelatedTo ? personRelatedTo.display : "";
             };
 
+            var getName = function (patient) {
+                return patient.givenName + (patient.middleName ? " " + patient.middleName : "") +
+                    (patient.familyName ? " " + patient.familyName : "");
+            };
+
+            var getPersonB = function (personName, personUuid) {
```

Comment:
`getPersonB` doesn't seem like an appropriate naming for a function. What does this do exactly?

Reply:
Hi @mks-d,

I talked with @binduak. She will rename to *getRelative* during the merge